Cargo.toml

```diff
@@ -25,4 +25,4 @@ bzip2 = ["librocksdb-sys/bzip2"]
 
 [dependencies]
 libc = "0.2"
-librocksdb-sys = { path = "librocksdb-sys", version = "6.6.4" }
+librocksdb-sys = { path = "librocksdb-sys", version = "6.6.5" }
```

Comment:
You didn't have to upgrade rocks itself.  We have another task for upgrading it to the new 6.8 branch.  I prefer you just revert these changes and stay on Rocksdb 6.6.4.  The only changes in this PR should be updating versions of dependent crates.